flask-redis/compose.yaml

```diff
@@ -0,0 +1,14 @@
+services:
+
```

Comment:
Remove blank line between `services` and `redis`.

Reply:
Did that. Should I also remove the blank line before `web`?

Reply:
No all good